### ISSUE TITLE
Using glz_custom_fields with MLP on TXP 4.3.0 creates a duplicate Excerpt field

### ISSUE THREAD

**Issue by johnstephens** [gerhard/glz_custom_fields_public#23]
Didn't see this on glz_cf forum thread or on the issue tracker: http://forum.textpattern.com/viewtopic.php?pid=239147#p239147

I upgraded a site from TXP 4.2.0 to TXP 4.3.0 today and the same thing happened to me: I see a duplicate Excerpt field that only appears when glz_cf is active. I can confirm that deactivating glz_custom_fields removes the duplicate Excerpt field.

**Comment by johnstephens** [comment]
Here's a temporary workaround. Add the following code to the bottom of your admin theme's stylesheet. The stylesheet is usually called `textpattern.css`, and it lives in the directory of the theme selected in Advanced Preferences:

```
/* !Hide Duplicate Excerpt Field
 * --------------------------------------------
 *  Bug introduced by upgrading to TXP v4.3.0
 *  with MLP & glz_cf active:
 *  https://github.com/gerhard/glz_custom_fields_public/issues#issue/23
 * --------------------------------------------
 */
p.excerpt {
display: none;
}
p.excerpt[dir] {
display: block;
}
```

**Comment by johnstephens** [comment]
After upgrading, I am seeing that this issue has cropped up again with the Body field, only this time the duplicate body field is the one with the `dir` attribute. <del>Here's my CSS fix:</del>

```
/* Edit: CSS fix does not work */
p.body[dir] {
    display: none;
}
```

<del>Shall I open a new issue?</del>


<ins>The above CSS cosmetically removes the duplicate body field, but Textpattern still picks up the content when you Publish or Save, which is a big problem when updating an article. I will open a new issue.</ins>

**Comment by gerhard** [comment]
John, did you receive my e-mail? Waiting for the details before I can sort this out...

**Comment by johnstephens** [comment]
I got it. I'll send you the info today.

**Comment by gerhard** [comment]
It's no rush, just wanted to make sure that we weren't waiting on one another.
